mocks/loader.go

```diff
@@ -406,6 +409,23 @@ func (l *Loader) loadHeaderIsConstraint(def map[interface{}]interface{}) (verifi
 	return newHeaderConstraint(header, valueStr, regexpStr)
 }
 
+func (l *Loader) loadBodyMatchesTextConstraint(def map[interface{}]interface{}) (verifier, error) {
+	var bodyStr, regexpStr string
+	if path, ok := def["body"]; ok {
+		bodyStr, ok = path.(string)
+		if !ok {
+			return nil, errors.New("`path` must be string")
+		}
+	}
+	if regexp, ok := def["regexp"]; ok {
+		regexpStr, ok = regexp.(string)
+		if !ok || regexp == "" {
+			return nil, errors.New("`regexp` must be string")
```

Comment:
Update error message, it must contain info about length of regexp field

Reply:
Other error messages about regex are similar. Removed comparison with an empty string.